### PR TITLE
Genomepy

### PR DESCRIPTION
**Pull request by simonvh**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pyfaidx/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   name: pyfaidx
-  version: "0.5.0"
+  version: {{ version }}
 
 source:
-  fn: pyfaidx-0.5.0.tar.gz
-  url: https://pypi.python.org/packages/6b/d5/18ce32ff5458c51c81e5e842dddf1e5be9ea1dd8877a484515b07f0a6201/pyfaidx-0.5.0.tar.gz
-  md5: 1e913f032303fad99b51fdbd98817a19
+  fn: pyfaidx-{{ version }}.tar.gz
+  url: https://github.com/mdshw5/pyfaidx/archive/v{{ version }}.tar.gz
```
Comment (mbargull) [inline recipes/pyfaidx/meta.yaml]: Could you use
```yaml
  url: https://pypi.io/packages/source/p/pyfaidx/pyfaidx-{{ version }}.tar.gz
  sha256: 7d5877fab01849fee78cdc7d336267310c07165171a099b4e024061c54dceb08
```
instead?

File: recipes/genomepy/meta.yaml [hunk 2]
```diff
@@ -49,8 +49,11 @@ test:
   imports:
     - genomepy
 
+  commands: 
+    - genomepy -h # [not py3k]
```
Comment (mbargull) [inline recipes/genomepy/meta.yaml]: If you use
```yaml
extra:
  container:
    # click requires a unicode locale when used with Python 3
    # extended-base generates en_US.UTF-8 locale and sets LC_ALL, LANG properly
    extended-base: true  # [py3k]
```
, you can re-enable this test.

Comment (simonvh) [reply recipes/genomepy/meta.yaml]: This is very useful to know, thanks @mbargull!

**Comment by mbargull** [comment]
Great, thanks!

**Pull request merged**